src/helpers/runSyncModules.js

```diff
@@ -20,11 +22,11 @@ export default function runSyncModules(config) {
     sync,
   } = config;
 
-  return promiseTry(() => sync.hooks.map((syncScriptPath) => {
-    const module = getSyncModule(config, syncScriptPath);
-    return module(config);
-  })).catch((error) => {
-    console.error(chalk.red(error));
-    process.exit(1);
-  });
+  return sync.hooks.reduce(
+    (prev, syncScriptPath) => prev.then(() => promiseTry(() => {
+      const syncModule = getSyncModule(config, syncScriptPath);
+      return syncModule(config);
+    })),
+    Promise.resolve(),
+  );
```

Comment:
we'll want a `.then(() => {})` as well, to hide the return value from the last hook

Reply:
added here: https://github.com/airbnb/react-component-variations/pull/15/files#diff-61076cd2353dba432496a9f09dc3048bR28